### ISSUE TITLE
about traing hyper-parameters of HICO-DET

### ISSUE THREAD

**Issue by dongsheng1991** [ASMIftekhar/VSGNet#16]
Hi, after reading the paper, I felt very confused about the training hyperparameters of HICO-DET. Could you please describe the specific hyperparameters in detail?  Thanks!

**Comment by ASMIftekhar** [comment]
Please check [scripts_hico/main.py](https://github.com/ASMIftekhar/VSGNet/blob/master/scripts_hico/main.py) for the hyperpaprameters. However, I think we did not use the scheduler mention there. We reduced our learning rate by 0.1 after 10 epochs.

**Comment by dongsheng1991** [comment]
Thank you for your reply！Is there any difference between the two schedulers on the performance of VSGNet?

**Comment by ASMIftekhar** [comment]
The best I can remember, the difference in performance was not really significant.

**Comment by dongsheng1991** [comment]
Thank you for your reply！I observe that in V-COCO the learning rate is increased by 10 after 10 epochs, while in HICO-DET the learning rate is reduced by 0.1. I am curious if there is any reason for this.

**Comment by ASMIftekhar** [comment]
Training in HICO-DET was really stable as it is a big dataset. So change in learning rate doesn't really matter that much. For V-COCO, the model converges quickly with this scheduler.

**Comment by dongsheng1991** [comment]
Thanks!
